### DISCUSSION TITLE
Window drag threshold

### DISCUSSION THREAD

**Discussion by NicholasSloan** [rxhanson/RectanglePro-Community#105] [Ideas]
Hi Ryan, any chance of a min distance setting for modifier drag and resize? This could reduce the danger of inadvertent operation when using common modifier combos. (BetterSnapTool has a similar "pixel threshold" setting.)
Many thanks for a great app.

**Comment by rxhanson** [comment]
Makes sense, I'll put it on my backlog and see if I can fit it into an upcoming release.
